GooglePlayInstant/Editor/QuickDeploy/AssetBundleVerifierWindow.cs

```diff
@@ -25,32 +29,44 @@ public class AssetBundleVerifierWindow : EditorWindow
     {
         private const int FieldMinWidth = 170;
 
-        private bool _assetBundleDownloadIsSuccessful;
-        private string _assetBundleUrl;
+        internal bool AssetBundleDownloadIsSuccessful;
+        internal string AssetBundleUrl;
+
         private long _responseCode;
-        private string _errorDescription;
+
+        internal string ErrorDescription;
+
         private string _mainScene;
         private double _numOfMegabytes;
         private UnityWebRequest _webRequest;
+        private AssetBundle _bundle;
+
+        public AssetBundleVerifyState State;
+
+        internal static string WebRequestErrorFormatMessage = "Problem retrieving AssetBundle from {0}: {1}";
+
+        public enum AssetBundleVerifyState
+        {
+            InProgress,
+            DestinationError,
+            WebRequestError,
+            BundleError,
+            DownloadSuccess
+        }
 
         /// <summary>
         /// Creates a dialog box that details the success or failure of an AssetBundle retrieval from a given assetBundleUrl.
         /// </summary>
-        public static void ShowWindow(string assetBundleUrl)
+        public static void ShowWindow()
         {
-            // Set AssetBundle url in a private variable so that information displayed in window is consistent with
-            // the url that this was called on. 
-            var window = (AssetBundleVerifierWindow) GetWindow(typeof(AssetBundleVerifierWindow), true,
-                "Play Instant AssetBundle Verify");
-            window._assetBundleUrl = assetBundleUrl;
-
-            window.StartAssetBundleVerificationDownload();
+            GetWindow(typeof(AssetBundleVerifierWindow), true, "Play Instant AssetBundle Verify");
```

Comment:
`return GetWindow(typeof(AssetBundleVerifierWindow), true, "Play Instant AssetBundle Verify");`

Reply:
Needs to be casted, but resolved.